<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateTileUploadEditionPageWizard</class>
 <widget class="QWizardPage" name="UpdateTileUploadEditionPageWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uploading form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_datastore">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Datastore:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DatastoreComboBox" name="cbx_datastore"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_stored_data">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dataset:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="StoredDataComboBox" name="cbx_stored_data"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="UploadCreationWidget" name="wdg_upload_creation" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatastoreComboBox</class>
   <extends>QComboBox</extends>
   <header>geotuileur.gui.cbx_datastore</header>
  </customwidget>
  <customwidget>
   <class>StoredDataComboBox</class>
   <extends>QComboBox</extends>
   <header>geotuileur.gui.cbx_stored_data</header>
  </customwidget>
  <customwidget>
   <class>UploadCreationWidget</class>
   <extends>QWidget</extends>
   <header>geotuileur.gui.upload_creation.wdg_upload_creation</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
